bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties

```diff
@@ -1,477 +1,53 @@
-
-# General messages
-message.versioncheck.failed = Firmware-Version konnte nicht geprüft werden: {0}
-message.versioncheck.beta = Es wurde eine Betaversion erkannt: {0}/{1}.
-message.versioncheck.tooold = ACHTUNG: Eine alte Firmware wurde erkannt: {0}/{1}, minimal erforderlich {2}.
-message.versioncheck.update = INFO: Eine neue Firmwareversion ist verfügbar, aktuell: {0}, neu: {1}
-message.versioncheck.autocoiot = INFO: Die Firmware unterstützt die Anforderung, Auto-CoIoT wurde aktiviert.
-message.init.noipaddress = Es konnte keine lokale IP-Adresse ermittelt werden. Bitte sicherstellen, dass IPv4 aktiviert ist und das richtige Interface in der openHAB Netzwerk-Konfiguration ausgewählt ist.
-message.init.protected = Das Gerät ist passwortgeschützt, die Zugangsdaten müssen in der Thing Konfiguration hinterlegt werden.
-message.command.failed = FEHLER: Der Befehl {0} für Kanal {1} kann nicht verarbeitet werden
-message.command.init = Thing aktuell nicht initialisiert, der Befehl {0} führt zur Initialisierung
-message.status.unknown.initializing = Initialisierung oder Gerät im Schlafmodus. 
+offline.conf-error-wrong-mode = Das Ger?t befindet sich nicht mehr in der erwarteten Betriebsart. L?schen Sie das Ger?t und f?hren Sie eine erneute Ger?teerkennung durch.
+offline.status-error-timeout = Das Ger?t ist nicht erreichbar (API Timeout).
+offline.status-error-unexpected-api-result = Es trat ein unerwartetes Problem beim API-Zugriff auf. ?berpr?fen Sie die Logdatei f?r genauere Informationen.
+offline.status-error-watchdog = Das Ger?t antwortet nicht und ist vermutlich nicht mehr verf?gbar.
+offline.status-error-restarted = Das Ger?t wurde neu gestartet und wird erneut initialisiert.
+offline.status-error-fwupgrade = Firmware-Update läuft
+
+message.versioncheck.failed = Firmware-Version kann nicht überprüft werden\: {0}
+message.versioncheck.beta = Gerät verwendet eine Beta-Version\: {0}/{1}.
+message.versioncheck.tooold = WARNUNG\: Firmware könnte zu alt sein, installiert\: {0}/{1}, Minimum {3} erforderlich.
+message.versioncheck.update = INFO\: Neue Firmware verfügbar\: aktuelle Version\: {0}, neue Version\: {1}
+message.versioncheck.autocoiot = INFO\: Die Firmware unterst?tzt die Anforderung, Auto-CoIoT wurde aktiviert.
+message.roller.calibrating = Das Ger?t ist nicht kalibriert. Es ist eine Kalibrierung mit der Shelly App erforderlich.
+message.init.noipaddress = Es konnte keine lokale IP-Adresse ermittelt werden. Bitte sicherstellen, dass IPv4 aktiviert ist und das richtige Interface in der openHAB Netzwerk-Konfiguration ausgew?hlt ist.
+message.init.protected = Das Ger?t ist passwortgesch?tzt, die Zugangsdaten m?ssen in der Thing Konfiguration hinterlegt werden.
+message.command.failed = FEHLER\: Der Befehl {0} f?r Kanal {1} kann nicht verarbeitet werden
+message.command.init = Thing aktuell nicht initialisiert, der Befehl {0} f?hrt zur Initialisierung
+message.status.unknown.initializing = Initialisierung oder Ger?t im Schlafmodus. 
```

Comment:
The German Shelly translations file is now messed up with ? values.

Reply:
I am afraid they are already messed up I. Crowdin. Someone has to fix them over there. I will give it a try tomorrow.

![Screenshot_20211228-205021~2](https://user-images.githubusercontent.com/5131747/147602120-8ced7ceb-6f13-4274-a61e-159c1409ec1c.jpg)

Reply:
Thanks for cleaning up the mess!